Playground/SwiftShortcutsTemplate.playgroundbook/Contents/Modules/ShortcutSupport.playgroundmodule/Sources/ShortcutPreviewViewController.swift

```diff
@@ -0,0 +1,132 @@
+import SwiftShortcuts
+import UIKit
+
+class ShortcutPreviewViewController<S: Shortcut>: UIViewController, UIDragInteractionDelegate {
+    let shortcut: S
+
+    var name: String {
+        return "\(S.self)"
+    }
+
+    init(shortcut: S) {
+        self.shortcut = shortcut
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        let rect = CGRect(origin: .zero, size: CGSize(width: 3.0, height: 3.0))
+        let render = UIGraphicsImageRenderer(size: rect.size)
+        let backgroundImage = render.image { context in
+            UIColor(red: 0.11, green: 0.12, blue: 0.34, alpha: 1.0).setFill()
+            UIRectFill(rect)
+        }
+        .resizableImage(withCapInsets: UIEdgeInsets(top: 1, left: 1, bottom: 1, right: 1))
+
+        let titleLabel = UILabel()
+        titleLabel.text = "Share \(name)"
+        titleLabel.font = .preferredFont(forTextStyle: .callout)
+        titleLabel.textColor = .white
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Why don't we use the `titleLabel` that already exists on `UIButton`?

Reply:
I wanted to adjust the `titleEdgeInsets` but once you do that then `UIButton`'s intrinsic content size is calculated incorrectly and causes text truncation... Since I'm using `UIButton.ButtonType.custom` I figured it was better just to add my own label and manage it's position with my own constraints